python/tvm/meta_schedule/relay_integration.py

```diff
@@ -97,13 +99,16 @@ def _normalize_params(
     if executor is None:
         executor = relay.backend.Executor("graph")
 
+    if runtime is None:
+        runtime = relay.backend.Runtime("cpp")
```

Comment:
May I ask if this change is necessary here and does it impact other target usage? If so can we add some comments to explain this runtime is for cpp runtime only?

Reply:
I need to pass the runtime to the `relay.build` for micro targets. I don't think it would affect other target usage, since the `Runtime("cpp")` is the default runtime in the `relay.build` anyway, I followed the example of how `executor` is set for the `runtime`. (see https://github.com/apache/tvm/blob/main/python/tvm/relay/build_module.py#L80)